drivers/sqlboiler-mysql/driver/mysql.go

```diff
@@ -60,13 +60,33 @@ func (m *MySQLDriver) Assemble(config drivers.Config) (dbinfo *drivers.DBInfo, e
 		}
 	}()
 
-	user := config.MustString(drivers.ConfigUser)
-	pass := config.MustString(drivers.ConfigPass)
-	dbname := config.MustString(drivers.ConfigDBName)
-	host := config.MustString(drivers.ConfigHost)
-	port := config.DefaultInt(drivers.ConfigPort, 3306)
-	sslmode := config.DefaultString(drivers.ConfigSSLMode, "true")
+	var (
+		user, pass, dbname, host string
+		port                     int
+	)
+
+	if dsn, ok := config.String(drivers.ConfigDSN); ok {
```

Comment:
Rather than looking up the DSN, parsing it, then assembling it into a DSN again, you should just assign `m.connStr = dsn`, half of the reason for taking the dsn directly is that there's features the config file doesn't allow you to provide, and you want to override it's choices.